app/views/messages/new.html.erb

```diff
@@ -1,24 +1,25 @@
-<div class="container-fluid" style="padding-top: 20px; padding-left:; width: 30%; float: centered;">
-  <div align="center">
-    <h1> Contact Form </h1>
-    <br>
-      <p>
-        Please send us feedback, comments, or any questions below.
-      </p>
-  </div>
-    <%= form_for @message, url: create_message_path do |f| %>
-
-      <%= @message.errors.full_messages.join(', ') %>
+<head>
+  <title>Contact</title>
+  <link href="form-validation.css" rel="stylesheet">
```

Comment:
The emails are not going through, you get this error:
ActionController::RoutingError (No route matches [GET] "/form-validation.css")
Make sure to see that the email message is outputted in the terminal when you press send.